driver/src/price_finding/price_finder_interface.rs

```diff
@@ -36,16 +37,50 @@ impl FromStr for SolverType {
             "standard-solver" => Ok(SolverType::StandardSolver),
             "fallback-solver" => Ok(SolverType::FallbackSolver),
             "naive-solver" => Ok(SolverType::NaiveSolver),
+            "open-solver" => Ok(SolverType::OpenSolver),
             _ => Err(anyhow!("solver type does not exit")),
         }
     }
 }
 
 impl SolverType {
-    pub fn to_args(self) -> Vec<String> {
+    pub fn to_args(self, result_folder: &str, input_file: &str, time_limit: String) -> Vec<String> {
+        let mut standard_solver_args: Vec<String> = vec![
+            String::from("-m"),
+            String::from("scripts.e2e._run"),
+            format!("{}{}", "/app/", input_file.to_owned()),
+            format!("--outputDir={}{}", "/app/", result_folder),
+            format!("--solverTimeLimit={}", time_limit),
+        ];
+        let open_solver_args = vec![
+            String::from("-m"),
+            String::from("src.match"),
+            format!("{}{}", "/app/", input_file.to_owned()),
+            format!(
+                "--solution={}{}{}",
+                "/app/",
+                result_folder.to_owned(),
+                "06_solution_int_valid.json",
+            ),
```

Comment:
Slightly off topic, but where is the source code for the open solver located?

Reply:
in the dir: /app/open_solver/ and then in the module src.match

Reply:
I don't see any `/app` directory. Is it in a different project somewhere? Is the repo public?

Reply:
https://github.com/gnosis/dex-open-solver

check out the docker file and you will see the app/